src/ToolboxBundle/Migrations/Version00000001.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace ToolboxBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Pimcore\Migrations\Migration\AbstractPimcoreMigration;
+
+class Version00000001 extends AbstractPimcoreMigration
+{
+    /**
+     * @return bool
+     */
+    public function doesSqlMigrations(): bool
```

Comment:
Pimcore 5.4 compatibility?

Reply:
true story. will do that in a separate pr. thanks!

Reply:
NVM - all return types are already available in 7.1 :)

http://php.net/manual/de/migration71.new-features.php

Reply:
no problem here, pimcore 5.4 requires php >=7.1 which supports return types like those.